application/config/config.php

```diff
@@ -288,74 +328,153 @@
 */
 $config['log_date_format'] = 'Y-m-d H:i:s';
 
+/*
+|--------------------------------------------------------------------------
+| Error Views Directory Path
+|--------------------------------------------------------------------------
+|
+| Leave this BLANK unless you would like to set something other than the default
+| application/views/errors/ directory.  Use a full server path with trailing slash.
+|
+*/
+$config['error_views_path'] = '';
+
 /*
 |--------------------------------------------------------------------------
 | Cache Directory Path
 |--------------------------------------------------------------------------
 |
 | Leave this BLANK unless you would like to set something other than the default
-| system/cache/ folder.  Use a full server path with trailing slash.
+| application/cache/ directory.  Use a full server path with trailing slash.
 |
 */
 $config['cache_path'] = '';
```

Comment:
Should this be set to the `shared` dir, similar to `archive` and `uploads`? Or do we need to do anything with dir permissions in datagov-deploy?

Reply:
I think you're right, it should just use the same directory. Will set that.

Reply:
On second thought, I'll leave it where it is since the directory already exists, but I will add to the Ansible to make it writable.